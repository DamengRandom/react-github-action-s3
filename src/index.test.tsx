describe('Test App', () => {
  it('should do snapshot test', () => {
    const RootDOM = document.querySelector('#root');
    expect(RootDOM).toMatchSnapshot();
  });
});
