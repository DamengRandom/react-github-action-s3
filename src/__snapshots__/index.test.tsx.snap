// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App should do snapshot test 1`] = `null`;
